### PR TITLE
Devonagent pro

### PR DESCRIPTION
**Pull request by nbsantos**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/devonagent-pro.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'devonagent-pro' do
+  version '3.9.4'
+  sha256 '1d3ca53676c55a228e3a0d87a9abd05d692b123bc65a1e9820f4186a2b065925'
+
+  # amazonaws.com/DTWebsiteSupport was verified as official when first introduced to the cask
```
Comment (adidalal) [inline Casks/devonagent-pro.rb]: Should be `s3.amazonaws.com/DTWebsiteSupport`

File: Casks/devonagent-pro.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  url "https://s3.amazonaws.com/DTWebsiteSupport/download/devonagent/#{version}/DEVONagent_Pro.app.zip"
+  appcast 'http://www.devon-technologies.com/fileadmin/templates/filemaker/sparkle.php?product=300005193&format=xml',
+          checkpoint: '9d696c00b3076dee96be1a5b441b3574fd0976031c0e5dcadeacf6fefbeedc84'
+  name 'DEVONagent Pro'
+  homepage 'http://www.devontechnologies.com/products/devonagent/devonagent-pro.html'
+  license :commercial
+
+  app 'DEVONagent.app', target: 'DEVONagent Pro.app'
```
Comment (adidalal) [inline Casks/devonagent-pro.rb]: Why the rename - is it possible to have multiple versions installed at the same time?

Comment (vitorgalvao) [inline Casks/devonagent-pro.rb]: Every rename **must** have [a commented reason using a standardised comment](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases).

[`devonthink-pro-office`](https://github.com/caskroom/homebrew-cask/blob/801219f8a3457ec1fba46038270fcf4b5712f31c/Casks/devonthink-pro-office.rb#L15L16) is a good example.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/devonagent-pro.rb (hunk 2 above)
Comment (nbsantos) [inline Casks/devonagent-pro.rb]: This is similar to the devonthink-pro-office.rb formula. For some reason the vendor messed-up the naming on the zip file.

Comment (vitorgalvao) [inline Casks/devonagent-pro.rb]: @nbsantos The comment is still needed. See the linked documentarion.

**Comment by fanquake** [comment]
@nbsantos Can you make the requested changes?

**Comment by vitorgalvao** [comment]
Closing for lack of reply. If you still wish to include this, please resubmit with the requested fixes.

**Pull request closed without merging**